e2e_tests/src/lib.rs

```diff
@@ -222,6 +222,32 @@ impl TestClient {
         )
     }
 
+    pub fn generate_ecc_key_pair(&mut self, key_name: String) -> Result<()> {
```

Comment:
I'm guessing this is meant strictly for exporting the key? You can probably rewrite it to do the whole process - generate, export, destroy

Reply:
Yeah, I'm not sure about all of these separate `generate`, `export`, etc. methods. I was copying the was its already done but I've ended up with a load of different methods. I guess the aim is to make them generic so they can easily be used in different tests.